script/Ciel/Day 10/10_01_CIEL10_9.txt

```diff
@@ -751,7 +751,8 @@ Are you saying I'm not contaminated?!
 [sha:902b4d50071a15718e21c3d035a0a6f7624a459f]{
 -- Page 645, Offset 28912.
 -- 「――――――！」
```

Comment:
This is Noel doing the horrified gasp; not sure how to render it lmao
>Hnnngh!

maybe?

Reply:
works for me